include/vrv/options.h

```diff
@@ -457,6 +457,7 @@ class Options {
     OptionBool m_landscape;
     OptionBool m_mensuralToMeasure;
     OptionDbl m_midiTempoAdjustment;
+    OptionDbl m_minLastJust;
```

Comment:
Maybe rename to `m_minLastJustification`? This will make it easier to understand what it is.